markhor_flippers/launch/flippers.launch

```diff
@@ -1,9 +1,8 @@
 <launch>
-    <arg name="robot_namespace" default="/markhor/flippers" />
-    
-    <param name="robot_description" command="xacro '$(find markhor_description)/urdf/markhor.urdf.xacro'" />
+    <arg name="robot_namespace" default="/markhor" />
+
     <!-- Load hardware interface-->
-    <node name="markhor_flippers_node" pkg="markhor_flippers" type="markhor_flippers_node" ns="$(arg robot_namespace)" output="screen">
+    <node name="markhor_flippers_node" pkg="markhor_flippers" type="markhor_flippers_node" ns="$(arg robot_namespace)/flippers" output="screen">
```

Comment:
Some changes need to be made in the flipper node. I am getting the following error when trying to test on the robot
![image](https://user-images.githubusercontent.com/46896242/214124058-9927a211-bf36-4d41-9313-38ec2884bc67.png)

Reply:
This is fixed in the last commit, we are now using 2 separate yaml file, flipper.yaml contains the parameters for the dummy position controller that is just a passthrough relic for the command, if the type was ever to be changed, we would summon a pid in line with the joystick command most certainly causing a runaway.